cordova/cordova-feature-android-tests/feature/comm.py

```diff
@@ -127,7 +127,7 @@ def installWebviewPlugin(pkg_mode, self):
     pluginstatus = commands.getstatusoutput(plugin_install_cmd)
     self.assertEquals(0, pluginstatus[0])
 
-def create(appname, pkgname, mode, sourcecodepath, replace_index_list, self):
+def create(appname, pkgname, mode, sourcecodepath, replace_index_list, self, actual_plugin = None):
```

Comment:
's/actual_plugin/extra_plugin', please update it in another PR, I will merge this one as hot-fix.